lib/error.js

```diff
@@ -39,6 +39,12 @@ var ERRNO = {
   BOUNCE_COMPLAINT: 133,
   BOUNCE_HARD: 134,
   BOUNCE_SOFT: 135,
+  EMAIL_EXISTS: 136,
+  EMAIL_DELETE_PRIMARY: 137,
+  SESSION_UNVERIFIED: 138,
+  USER_PRIMARY_EMAIL_EXISTS: 139,
+  VERIFIED_PRIMARY_EMAIL_EXISTS: 140,
+  UNVERIFIED_PRIMARY_EMAIL_NEWLY_CREATED: 141,
```

Comment:
Heh definitely a little wordy but couldn't think of a better name. It is thrown [here](https://github.com/mozilla/fxa-auth-server/blob/7558d7be2454c30b5a063fe84f69473c5328c82d/lib/routes/account.js#L2093). Part of the acceptance criteria is to not allow secondary email to be added if there was a newly created account (less than 24hrs old) using that email.